openeogeotrellis/integrations/calrissian.py

```diff
@@ -199,7 +199,7 @@ def create_input_staging_job_manifest(self, cwl_source: CwLSource) -> Tuple[kube
 
         container = kubernetes.client.V1Container(
             name="calrissian-input-staging",
-            image="alpine:3",
+            image="docker-upstream.artifactory.vgt.vito.be/alpine:3.17.0",
```

Comment:
I think we should avoid putting `artifactory.vgt.vito.be`  references in openeo-geopyspark-driver code

Reply:
Would something like docker://registry.stag.warsaw.openeo.dataspace.copernicus.eu/rand/alpine:3 be better?

Reply:
Ideally, we just have an "abstract" `alpine:3` reference here, but there is a way to configure the underlying k8s machinery to resolve this from the desired docker repository (e.g. like you can have global `pip` config files to set a prefered "index-url" instead of the default pypi.org)


Alternatively, we introduce a config field here so it can be override by a deployment